index.html

```diff
@@ -95,84 +95,7 @@ <h2 class="mobile-work-header">My Recent </br> Works</h2>
         <h2 class="desktop-work-header">My Recent Works</h2>
     </div>
     <div class="myworks-wrapper">
```

Comment:
- @demesameneshoa Kudos on implementing the popup with dynamic data using javascript, how ever i notice the project images popup on the mobile screen look reduced, compared to the figma design, please apply the necessary styles to fix this.

 **Advise**

-  I will advise you to add a scrollable on the modal popup in case its height goes above 100vh so the user can scroll to see the rest of the details on the modal.

##### Your design
![image](https://github.com/demesameneshoa/Portfolio/assets/58771507/d9fc7656-f033-4e91-9a2c-3321bc586194)

##### Figma design
![image](https://github.com/demesameneshoa/Portfolio/assets/58771507/383db6dd-a1b2-4705-a016-28311e875b1a)